src/Text/Pandoc/Writers/HTML.hs

```diff
@@ -557,13 +557,7 @@ footnoteSection refLocation startCounter notes = do
          container $ do
            nl
            hrtag
-           -- Keep the previous output exactly the same if we don't
-           -- have multiple notes sections
-           if startCounter == 1
```

Comment:
I've assumed that this can be dropped as long as ordered lists are not used anymore, is it really the case?

Reply:
No, it can't be dropped. This plays a significant role when you use `--reference-location=block` (or `section`), which causes the footnotes to be printed at the end of each block (or section).  In that case we need to make sure that the numbering continues appropriately in each new footnote section.